wls-exporter-core/src/main/java/com/oracle/wls/exporter/ExporterCall.java

```diff
@@ -63,6 +61,19 @@ private void displayMetrics(WebClient webClient, MetricsStream metricsStream, MB
     }
   }
 
+  private void reportProblem(MetricsStream metricsStream, MBeanSelector selector) {
+    metricsStream.println(withCommentMarkers(getProblem(selector) + "\n" + selector.getPrintableRequest()));
+  }
+
+  private String getProblem(MBeanSelector selector) {
+    if (selector.isRequestForPrivilegedProperty())
+      return "You seem to have encountered a bug in the WebLogic REST API.\n" +
```

Comment:
Can this text come from a resource bundle.

Reply:
It could; actually, it could probably be removed, since I know have altered the code to exclude it.